rust/flatbuffers/src/vector.rs

```diff
@@ -42,6 +42,10 @@ impl<'a, T: 'a> Vector<'a, T> {
     pub fn len(&self) -> usize {
         read_scalar::<UOffsetT>(&self.0[self.1 as usize..]) as usize
     }
+    #[inline(always)]
```

Comment:
@CasperN Still not sure why this `is_empty` function should be part of the PR?

Reply:
That's one of the cargo clippy lints, they suggest you have `is_empty` if you have `len`

https://rust-lang.github.io/rust-clippy/master/#len_without_is_empty

Reply:
SGTM